### PR TITLE
Add docs on hydrus

### PR DESCRIPTION
**Pull request by sameshl**
### Description

Add docs on `hydrus` explaining how `hydrus` goes about creating the database(and the server) when given an API Doc.

### CODE REVIEW

File: docs/source/hydrus.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+-  The table for a collection is slightly different from a parsed_class.
+   Any collection will have only 3 columns. Those are ``id``\ (primary
+   key), ``members``\ (this is used for storing the ``id`` of the
+   instance of the class that the collection *manages*) and
+   ``collection_id`` which is the id for any collection the user
+   creates. **Note:** For now, ``hydrus`` only supports a collection to
+   hold references to only a single type of class.
+
+      A collection is defined as a "set of somehow related resources".
```
Comment (chrizandr) [inline docs/source/hydrus.rst:61]: use hydra:Link instead of the word references.

**Comment by chrizandr** [comment]
Shouldn't this be in hydrus? Why are we adding docs about the database to core?

File: docs/source/hydrus.rst (hunk 1 above)
Comment (sameshl) [reply docs/source/hydrus.rst:61]: Could you elaborate on what you mean @chrizandr ? I did not get you :sweat_smile:

**Comment by sameshl** [comment]
> Shouldn't this be in hydrus? Why are we adding docs about the database to core?

This was discussed in the last call, right? You had said to add general docs on `hydrus` in the core module?

**Comment by chrizandr** [comment]
We had discussed adding the databse changes to hydrus and the core changes to core. You don't have to mention anything about the hydrus database in the core docs. Core should be a standalone library as is.

**Comment by sameshl** [comment]
> We had discussed adding the databse changes to hydrus and the core changes to core. You don't have to mention anything about the hydrus database in the core docs. Core should be a standalone library as is.

Shifted this PR to https://github.com/HTTP-APIs/hydrus/pull/495

**Pull request closed without merging**